src/main/include/subsystems/Shooter.h

```diff
@@ -48,6 +52,8 @@ namespace t34
 
         void SetMaxSpeedPercent(const double percent);
```

Comment:
Instead of using magic number in Robot.cpp, what do you think about adding methods to the Shooter that make the code a little more self-documenting?

```suggestion
        void SetMaxSpeedPercent(const double percent);
        
        inline void SetMaxSpeedForAmp() { SetMaxSpeedPercent(0.1); }
        inline void SetMaxSpeedForSpeaker() { SetMaxSpeedPercent(0.4); }
        inline void SetMaxSpeedForTrap() { SetMaxSpeedPercent(0.7); }
        inline void SetMaxSpeedForNoteCollection() { SetMaxSpeedPercent(0.0); }
```

Then the code in Robot.cpp could look like this:

```cpp
        case (90): // speaker - right
            rc->shooter.SetMaxSpeedForSpeaker();
            // ...
            break;
```